ert_shared/ensemble_evaluator/prefect_ensemble/storage_driver.py

```diff
@@ -0,0 +1,36 @@
+import os
+import shutil
+
+
+def storage_driver_factory(config, run_path):
+    if config.get("type") == "shared_disk":
+        storage_path = config["storage_path"]
+        return SharedDiskStorageDriver(storage_path, run_path)
+    else:
+        raise ValueError(f"Not a valid storage type. ({config.get('type')})")
+
+
+class SharedDiskStorageDriver:
```

Comment:
It appears to me that the factory-function is the intended way of instantiating storage. If so, I think the class itself should be made private...